jme3-desktop/src/main/java/com/jme3/system/JmeDesktopSystem.java

```diff
@@ -337,7 +337,9 @@ public void initialize(AppSettings settings) {
             return;
         }
         initialized = true;
-        logger.log(Level.INFO, getBuildInfo());
+        if (logger.isLoggable(Level.INFO)) {
+            logger.log(Level.INFO, getBuildInfo());
```

Comment:
I think this is okay because `getBuildInfo()` has no side effects.

Reply:
However, since `JmeDesktopSystem` only gets initialized once, this is probably not a worthwhile change.

Reply:
About these 

> not a worthwhile change

They are already done, so there shouldn't be any harm? One millisecond faster app start up time in production with less logging :) Sounds wasteful to me if this comment means that the change should be reverted. Just kinda asking for clarification what exactly is requested by this comment.

Reply:
I'm requesting that the "not worthwhile" changes be reverted.

Reply:
Ok those changes have been reverted.